src/middlewared/middlewared/plugins/activedirectory.py

```diff
@@ -1563,7 +1566,7 @@ def __init__(self, **kwargs):
         self.middleware = kwargs.get('middleware')
         self.logger = self.middleware.logger
         self.finished = threading.Event()
-        self.state = MSG_WINBIND_ONLINE
+        self.state = 1027
```

Comment:
nit: would it make sense to append `# MSG_WINBIND_ONLINE` here, as you did below on `directoryservices.py` line 32 ?